plugin/src/main/groovy/grails/plugin/springsecurity/acl/ProxyAwareParameterNameDiscoverer.groovy

```diff
@@ -28,7 +28,7 @@ import java.lang.reflect.Method
  * @author <a href='mailto:burt@burtbeckwith.com'>Burt Beckwith</a>
  */
 @CompileStatic
-class ProxyAwareParameterNameDiscoverer extends LocalVariableTableParameterNameDiscoverer {
+class ProxyAwareParameterNameDiscoverer extends DefaultParameterNameDiscoverer {
```

Comment:
I do not know if `DefaultParameterNameDiscoverer` is a suitable replacement for `LocalVariableTableParameterNameDiscoverer`.  I do not know the purpose of this class.

Reply:
This was indeed problematic.  This class removed altogether and will just use Spring's default discoverer instead.